### PR TITLE
Set default modal title

### PR DESCRIPTION
**Pull request by kjuej**

### CODE REVIEW

**Comment by vladimyr** [comment]
> The issue seems to be that the `default` is not applied if the `title` is equal to an empty string.

Yup, vue will fall back to default only if prop receives `undefined`.

> 2\. Remove `title` from the `props`

This is _adventurous_ and le me explain why. If you use native title attribute there is no way to prevent/hide native tooltip being displayed on hover that is simply undesirable sometimes. I believe safer (albeit less elegant 😢 ) way would be using computed property 🙃

**Comment by abasic** [comment]
@vladimyr @kronicker Can you please take a look?

File: src/teaching-element/Modal/index.vue [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
   },
-  watch: {
-    showDialog(val) {
-      if (val) return document.body.classList.add('modal-open');
-      return document.body.classList.remove('modal-open');
+  methods: {
+    toggleDialog() {
+      this.showDialog = !this.showDialog;
+      const action = this.showDialog ? CLASS_ACTIONS.ADD : CLASS_ACTIONS.REMOVE;
+      bodyClassList[action](MODAL_OPEN_CLASS);
```
Comment (vladimyr) [inline src/teaching-element/Modal/index.vue]: 💡 Proposal:
```js
toggleDialog(state = !this.showDialog) {
  document.body.classList('modal-open', state);
  this.showDialog = state;
}
```

File: src/teaching-element/Modal/index.vue [hunk 2]
```diff
@@ -1,37 +1,39 @@
...
-  data() {
-    return { showDialog: false };
-  },
+  data: () => ({ showDialog: false }),
   computed: {
-    elements() {
-      return sortBy(this.embeds, 'position');
-    }
+    elements: ({ embeds }) => sortBy(embeds, 'position'),
+    buttonLabel: ({ title }) => title || DEFAULT_LABEL
```
Comment (vladimyr) [inline src/teaching-element/Modal/index.vue]: I know it sounds awful but it is actually a button text 🙃

**Pull request merged**